unpin.js

```
@@ -0,0 +1,44 @@
+const axios = require('axios');
+
+function deletePin(hashToUnpin){
+  const url = `https://api.pinata.cloud/pinning/unpin/${hashToUnpin}`;
+    return axios
+        .delete(
+            url,
+            {
+                headers: {
+                    'pinata_api_key': process.env.IPFS_DEPLOY_PINATA__API_KEY,
+                    'pinata_secret_api_key': process.env.IPFS_DEPLOY_PINATA__SECRET_API_KEY
+                }
+            }
+        ).then(function (response) {
+          //console.log(response);
+            //handle response here
+        })
+        .catch(function (error) {
+            //handle error here
+        });
+}
+
+function unPinExceptLast() {
+    const url = `https://api.pinata.cloud/data/pinList?status=pinned&pageOffset=1`;
+    return axios
+        .get(url, {
+            headers: {
+                'pinata_api_key': process.env.IPFS_DEPLOY_PINATA__API_KEY,
+                'pinata_secret_api_key': process.env.IPFS_DEPLOY_PINATA__SECRET_API_KEY
+            }
+        })
+        .then(function (response) {
+          response.data.rows.forEach(function(item){
+            deletePin(item.ipfs_pin_hash)
+          });
+          //console.log(response.data.rows);
+            //handle response here
+        })
+        .catch(function (error) {
+            //handle error here
+        });
```

```suggestion
```

Suggesting not to swallow possible pinata errors, so that build fails if unpinning fails